packages/core/test-e2e/insured-bridge/e2e_InsuredBridge.js

```diff
@@ -0,0 +1,118 @@
+const { ZERO_ADDRESS } = require("@uma/common");
+
+const { assert } = require("chai");
+
+const { ethers } = require("hardhat");
+const { Watcher } = require("@eth-optimism/watcher");
+const { getContractInterface, predeploys } = require("@eth-optimism/contracts");
+
+const {
+  createLocalEthersFactory,
+  createOptimismEthersFactory,
+  getProviders,
+} = require("./helpers/MultiProviderHelper");
+
+const { DEFAULT_ADMIN_KEY } = require("./helpers/OptimismConstants");
+
+// Create contract factories using L1 and L2 artifacts
+const factory__L1_ERC20 = createLocalEthersFactory("ExpandedERC20");
+const factory__L2_ERC20 = createOptimismEthersFactory("L2StandardERC20", true);
```

Comment:
OOC, why couldn't you use the same ExpandedERC20 ABI on L2?

Reply:
few points:
- 1 this does not return an ABI, it returns ABI+bytecode
- the bytecode for the L2 token is compiled to work on optimism (not L1) and so it wont work on between networks.
- the `L2StandardERC20` has permissions to be `mint` and `burn` from an `onlyL2Bridge` modifier. i.e it the bridge contract on L2 can create and destroy tokens on L2.
- the `L2StandardERC20` implements `supportsInterface` which the bridge uses to validate the correctness of the token.
- in production, the bridge will use `OVM_L2StandardTokenFactory` to mint new instances of this token. By us deploying like this we get the exact same contract as production would use on L2.